gtkm/auth/service_auth.py

```diff
@@ -1,13 +1,81 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Depends, HTTPException, Response, status
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import RedirectResponse, JSONResponse
+from sqlalchemy.orm import Session
+import requests
+import uuid
 
+from .database.database import Base, engine, get_db
+from .database.user_schema import User
+from .database.database_operations import get_user, get_user_by_github_login, create_user
+from .sessions.session import backend, cookie, verifier
+from .sessions.session_data import SessionData
+
+
+Base.metadata.create_all(bind=engine)
 auth = APIRouter()
 
+ROOT_URL = "http://127.0.0.1:8000"
```

Comment:
Czy to na pewno jest potrzebne?

Reply:
Usunąłem tę stałą - faktycznie jest niepotrzebna.